elements/pf-progress/pf-progress.ts

```diff
@@ -6,6 +6,7 @@ import { ifDefined } from 'lit/directives/if-defined.js';
 import { styleMap } from 'lit/directives/style-map.js';
 
 import styles from './pf-progress.css';
+import { InternalsController } from '@patternfly/pfe-core/controllers/internals-controller.js';
```

Comment:
This thing is pretty experimental, might be better to roll our own here, for now